### PR TITLE
Update Traditional Chinese Strings

### PR DESCRIPTION
**Pull request by tony8077616**
Update Traditional Chinese Strings

### CODE REVIEW

File: ShareX/Properties/Resources.zh-TW.resx [hunk 1]
```diff
@@ -1,530 +1,654 @@
...
+    <value>視窗標題</value>
+  </data>
+  <data name="ClipboardContentViewer_ClipboardContentViewer_Load_Clipboard_content__Image__Size___0_x_1__" xml:space="preserve">
+    <value>剪貼簿內容：檔案（尺寸： {0}x{1}）</value>
+  </data>
+  <data name="FFmpeg_FFmpeg_video_and_audio_source_both_can_t_be__None__" xml:space="preserve">
+    <value>FFmpeg 視訊和音訊來源兩者不可以為 「None」。</value>
+  </data>
+  <data name="InspectWindow_ProcessIdentifier" xml:space="preserve">
+    <value>程序識別器</value>
```
Comment (toto6038) [inline ShareX/Properties/Resources.zh-TW.resx:613]: 同列546之留言

**Review by toto6038 — CHANGES_REQUESTED** [review]

**Comment by Jaex** [comment]
Why there is so many deleted/replaced lines in this PR? There should be only added lines, see example: #6006

**Comment by brlin-tw** [comment]
I agree with @Jaex, maybe the line endings have been changed for the entire file?

@tony8077616 Can you fix it?

**Comment by tony8077616** [comment]
> Why there is so many deleted/replaced lines in this PR? There should be only added lines, see example: #6006

Hello sir 
I don't know why there are so many deleted/replaced with this file

I just...
1. Download the latest standalone version with that translate Application
2. Copy the ShareX from github and input that DIR to translate Application
3. Output Excel file from Translate Application
4. Add with translate and input excel file to translate Application
5. Get the update file in Properties and upload to github

So I don't know which step is the wrong...

File: ShareX/Properties/Resources.zh-TW.resx [hunk 2]
```diff
@@ -1,530 +1,654 @@
...
+    <value>描述</value>
+  </data>
+  <data name="ShareXIsMinimizedToTheSystemTray" xml:space="preserve">
+    <value>最小化 ShareX 到系統工作列圖示。</value>
+  </data>
+  <data name="AboutForm_AboutForm_Language_pt_PT" xml:space="preserve">
+    <value>葡萄牙語</value>
+  </data>
+  <data name="InspectWindow_ProcessName" xml:space="preserve">
+    <value>程序名稱</value>
```
[the remark this answers is not in the document]
Comment (tony8077616) [reply ShareX/Properties/Resources.zh-TW.resx:546]: 我是取自工作管理員的“處理程序”的部分，“處理序“還會給別人有缺字的狀況

[the remark this answers is not in the document]
Comment (tony8077616) [reply ShareX/Properties/Resources.zh-TW.resx:546]: 不然…
就乾脆直接叫做處理程序名稱好了

**Comment by tony8077616** [comment]
> > Why there is so many deleted/replaced lines in this PR? There should be only added lines, see example: #6006
> 
> Hello sir 
> I don't know why there are so many deleted/replaced with this file
> 
> I just...
> 1. Download the latest standalone version with that translate Application
> 2. Copy the ShareX from github and input that DIR to translate Application
> 3. Output Excel file from Translate Application
> 4. Add with translate and input excel file to translate Application
> 5. Get the update file in Properties and upload to github
> 
> So I don't know which step is the wrong...

I'll try to check out again in my free time ,may be tomorrow night if I don't forget

**Comment by tony8077616** [comment]
@Jaex I've already done for output
@brlin-tw Done for fixed the strings

**Pull request closed without merging**